src/libs/ReportUtils.ts

```diff
@@ -4618,6 +4625,61 @@ function shouldDisableThread(reportAction: OnyxEntry<ReportAction>, reportID: st
     );
 }
 
+function getAllAncestorReportActions(
```

Comment:
Please add a unit test for this method and some test cases to show the correct ancestor reportActions are returned.